app/workers/rapid_ready_for_decision/form526_hypertension_job.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'lighthouse/veterans_health/client'
+
+module RapidReadyForDecision
+  class Form526HypertensionJob < Form526BaseJob
+    STATSD_KEY_PREFIX = 'worker.fast_track.form526_hypertension_job'
+
+    sidekiq_options retry: 2
+
+    def assess_data(form526_submission)
+      assessed_data = query_and_assess_lighthouse(form526_submission)
+
+      return if assessed_data[:bp_readings].blank?
+
+      assessed_data
+    end
+
+    private
+
+    def query_and_assess_lighthouse(form526_submission)
+      client = lighthouse_client(form526_submission)
+      bp_readings = assess_bp_readings(client.list_resource('observations'))
+      # stop querying if bp_readings.blank?
+      medications = assess_medications(client.list_resource('medications')) if bp_readings.present?
```

Comment:
Other RRD jobs can split this into separate `query_lighthouse` and `assess_lighthouse_data` methods. For this class, since we want to stop querying if the assessed bp_readings is blank, we do the querying and assessing in a single method.